include/sys/zio_compress.h

```diff
@@ -107,15 +171,26 @@ extern int lz4_decompress_zfs(void *src, void *dst, size_t s_len, size_t d_len,
     int level);
 extern int lz4_decompress_abd(abd_t *src, void *dst, size_t s_len, size_t d_len,
     int level);
+extern int zstd_init(void);
+extern void zstd_fini(void);
```

Comment:
These don't need to be in the header anymore.

Reply:
wrong. its still referenced by libzpool. your argument is only true for the kernel implementation which uses module_init/exit. spa_misc.c still calls zstd_init and zstd_exit out from the userspace implementation. 

        metaslab_alloc_trace_init();
        ddt_init();
        zio_init();
#ifndef _KERNEL
        zstd_init();
#endif
        dmu_init();
        zil_init();
        vdev_cache_stat_init()